### ISSUE TITLE
Incorrect default value for 'setup_archive' attribute for Web Adaptor 10.5.1

### ISSUE THREAD

**Issue by pbobov** [Esri/arcgis-cookbook#104]
I've got a question related to the latest release of Esri's Cookbook 3.1.0: I'm trying to install ArcGIS Server 10.5.1 together with Web Adaptor for IIS. My JSON config file looks like this:
{
  "arcgis": {
    "version": "10.5.1",
    "repository": {
      "archives": "\\\\MyPath\\Software\\ESRI\\ArcGIS Enterprise 10.5.1",
      "setups": "D:\\AutomatedDeployment\\ArcGIS",
      "patches": "D:\\AutomatedDeployment\\ArcGIS\\Patches"
    },
    …
}

When executing the cookbook, it crashes during the installation of the Web Adaptor with the following exception:
[2017-08-15T13:50:09+02:00] FATAL: Mixlib::ShellOut::ShellCommandFailed: arcgis_enterprise_webadaptor[Install Web Adaptor for Server] (arcgis-enterprise::server_wa line 38) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '1'
---- Begin output of "D:\AutomatedDeployment\ArcGIS/ArcGIS 10.5.1/WebAdaptorIIS/Setup.exe" /qb VDIRNAME=arcgis ----
STDOUT:
STDERR: The system cannot find the path specified.
---- End output of "D:\AutomatedDeployment\ArcGIS/ArcGIS 10.5.1/WebAdaptorIIS/Setup.exe" /qb VDIRNAME=arcgis ----
Ran "D:\AutomatedDeployment\ArcGIS/ArcGIS 10.5.1/WebAdaptorIIS/Setup.exe" /qb VDIRNAME=arcgis returned 1

I've already found the weak point, it is located in C:\Chef\cookbooks\arcgis-enterprise\attributes\webadaptor.rb:
default['arcgis']['web_adaptor'].tap do |web_adaptor|

  web_adaptor['admin_access'] = false
  web_adaptor['install_system_requirements'] = true

  case node['platform']
  when 'windows'
    web_adaptor['setup'] = ::File.join(node['arcgis']['repository']['setups'],
                                       'ArcGIS ' + node['arcgis']['version'],
                                       'WebAdaptorIIS', 'Setup.exe')
    web_adaptor['lp-setup'] = node['arcgis']['web_adaptor']['setup']
    web_adaptor['install_dir'] = ''

    case node['arcgis']['version']
    when '10.5.1'
      web_adaptor['setup_archive'] = ::File.join(node['arcgis']['repository']['archives'],
                                            'Web_Adaptor_Java_Windows_1051_156379.exe')
      web_adaptor['product_code'] = '{0A9DA130-E764-485F-8C1A-AD78B04AA7A4}'
      web_adaptor['product_code2'] = '{B8A6A873-ED78-47CE-A9B4-AB3192C47604}'
    when '10.5'
      web_adaptor['setup_archive'] = ::File.join(node['arcgis']['repository']['archives'],
                                            'Web_Adaptor_for_Microsoft_IIS_105_154007.exe')
      web_adaptor['product_code'] = '{87B4BD93-A5E5-469E-9224-8A289C6B2F10}'
      web_adaptor['product_code2'] = '{604CF558-B7E1-4271-8543-75E260080DFA}'

Is there any reason why the installed version of Web Adaptor has changed between 10.5.0 and 10.5.1 (IIS vs. Java)?

**Comment by mschoelen** [comment]
@pbobov Did you ever figure out what parameters to specify to avoid this error? I'm encountering the same thing, notably because Chef is skipping the unpacking process for Web Adaptor.

```
Generated at 2018-01-25 13:26:25 -0500
Mixlib::ShellOut::ShellCommandFailed: arcgis_enterprise_webadaptor[Install Web Adaptor for Portal] (arcgis-enterprise::portal_wa line 38) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '1'
---- Begin output of "C:\Users\username\Documents/ArcGIS 10.5.1/WebAdaptorIIS/Setup.exe" /qb VDIRNAME=geo ----
STDOUT: 
STDERR: The system cannot find the path specified.
---- End output of "C:\Users\username\Documents/ArcGIS 10.5.1/WebAdaptorIIS/Setup.exe" /qb VDIRNAME=geo ----
Ran "C:\Users\username\Documents/ArcGIS 10.5.1/WebAdaptorIIS/Setup.exe" /qb VDIRNAME=geo returned 1
```

From PowerShell (showing that unpacking was skipped):

```
  * arcgis_enterprise_webadaptor[Unpack ArcGIS Web Adaptor] action unpack[2018-01-25T13:31:57-05:00] INFO: Processing ar
cgis_enterprise_webadaptor[Unpack ArcGIS Web Adaptor] action unpack (arcgis-enterprise::portal_wa line 20)
 (skipped due to only_if)
  * arcgis_enterprise_webadaptor[Install Web Adaptor for Portal] action install[2018-01-25T13:31:57-05:00] INFO: Process
ing arcgis_enterprise_webadaptor[Install Web Adaptor for Portal] action install (arcgis-enterprise::portal_wa line 38)


    ================================================================================
    Error executing action `install` on resource 'arcgis_enterprise_webadaptor[Install Web Adaptor for Portal]'
    ================================================================================
```

**Comment by pbobov** [comment]
The problem is fixed in arcgis-enterprise cookbook v3.2.0. 

```ruby
     web_adaptor['setup_archive'] = ::File.join(node['arcgis']['repository']['archives'],
                                            'Web_Adaptor_for_Microsoft_IIS_1051_156367.exe')
```
